frontend/src/data/config/MetadataMap.ts

```diff
@@ -74,6 +62,19 @@ const dataSourceMetadataList: DataSourceMetadata[] = [
     ],
     downloadable: false,
   },
+  {
+    id: "uhc",
+    data_source_name: "America's Health Rankings",
+    data_source_link:
+      "https://www.americashealthrankings.org/explore/annual/measure/Overall_a/state/ALL",
+    geographic_level: "State",
+    demographic_granularity: "Race/ethnicity, age, sex",
+    update_frequency: "Annual",
+    description:
+      "The prevalence (percentage) for diseases broken down by self-reported race/ethnicity and sex at the U.S. and state levels.",
```

Comment:
Should say "by self-reported race/ethnicity, sex, and age"

Reply:
right, thanks